pkg/lib/bundle/generate.go

```diff
@@ -114,13 +129,19 @@ func GetMediaType(directory string) (string, error) {
 	}
 
 	// Validate the file names to determine media type
-	for _, file := range files {
-		if file == "Chart.yaml" {
+	for fileName, fileBlob := range files {
+		// TODO: be more robust here, we should validate the format of helm charts
```

Comment:
I think the `Chart.yaml` filename is the only defining feature of a helm chart. I'll see if I can find something else to look for and update here if I do.

Reply:
There's a helm library to validate a chart. Here's an example from SDK:

https://github.com/operator-framework/operator-sdk/blob/ba68281353e529aefadb0c86c74979dd2cf69f12/pkg/helm/watches/watches.go#L83-L85